fuel-core/src/database.rs

```diff
@@ -392,7 +391,8 @@ mod relayer {
             _height: BlockHeight,
         ) -> Option<Arc<SealedFuelBlock>> {
             // TODO
-            Some(Arc::new(SealedFuelBlock::default()))
+            // Some(Arc::new(SealedFuelBlock::default()))
```

Comment:
Also blocked by #673

Reply:
Although this is no longer needed by the relayer correct?